src/test/scala/io/qbeast/spark/index/FailingTests.scala

```diff
@@ -0,0 +1,129 @@
+package io.qbeast.spark.index
+
+import io.qbeast.TestClasses.{Client3, T1, T2, TestStrings}
+import io.qbeast.core.model.{IndexStatus, QTableID}
+import io.qbeast.spark.QbeastIntegrationTestSpec
+import org.apache.spark.sql.{Dataset, SparkSession}
+
+class FailingTests extends QbeastIntegrationTestSpec with IndexTestChecks {
```

Comment:
These tests need to be renamed.

Reply:
The tests were already in `TransformerIndexingTest`, the only difference is that in `FailingTests` they had a small `cubeSize` set.
In 8cc6a4b I've added the. smallCubeSize test to `IndexTest`, but only for a base case; I don't think we need to test indexing different data types in this case.